AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLRequest/GraphQLRequest+AnyModelWithSync.swift

```diff
@@ -64,6 +64,39 @@ extension GraphQLRequest: ModelSyncGraphQLRequestFactory {
                                                    decodePath: document.name)
     }
 
+    public static func createMutation(of model: Model, version: Int?) -> GraphQLRequest<MutationSyncResult> {
+        return createMutation(of: model, modelSchema: model.schema, version: version)
```

Comment:
nit: remove redundant `return`
```suggestion
        createMutation(of: model, modelSchema: model.schema, version: version)
```